### ISSUE TITLE
[BUG] facing the mouse on harm intent is bound to the 1-4 button presses rather than the intent itself

### ISSUE THREAD

**Issue by BarteG44** [shiptest-ss13/Shiptest#2083]
## Reproduction:

-press 4 to toggle harm intent
-click on any other intent in the bottom right ui menu instead of using the 1-3 hotkeys

https://github.com/shiptest-ss13/Shiptest/assets/105491762/aeb5e09e-58da-4ae0-92e5-268dd23d190a

**Comment by thgvr** [comment]
yeah. I really gotta find a better way to do this - intents are shit code
